tests/puppeteer/campaigns/data/demo/tax.js

```diff
@@ -1,8 +1,9 @@
 module.exports = {
   DefaultFrTax: {
-    id: '1',
+    id: 1,
     name: 'TVA FR 20%',
-    rate: '20',
+    frName: 'FR Taux standard (20%)',
```

Comment:
What's the difference between `name` and `frName` ?

Reply:
name is English name (default one in our installation language : en), and frName is french name (added with french localization pack).

Reply:
Would it be possible in the future to have more than X languages? Like fr, pl..?

If yes, then you should probably provide an array there to improve extensibility

Reply:
I don't think so, they will be 2 names only. the name displayed in taxes grid and the one displayed in Gift-wrapping select in gift options.

Reply:
Alright then